tests/framework/enrolment-providers/class-sensei-test-enrolment-provider-provides-for-dinosaurs.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * File containing the class Sensei_Test_Enrolment_Provider_Provides_For_Dinosaurs.
+ *
+ * @package sensei-tests
+ */
+
+/**
+ * Class Sensei_Test_Enrolment_Provider_Provides_For_Dinosaurs.
+ *
+ * Used in testing. Provides enrolment for dinosaurs only.
```

Comment:
Not very inclusive @jom...

Reply:
![jp-newman-laughs](https://user-images.githubusercontent.com/68693/72295167-e2e18180-364e-11ea-816b-0452b5244821.gif)